.github/workflows/ci.yml

```diff
@@ -1,17 +1,35 @@
 name: CI
 
 on:
-  push:
-    branches:
-      - main
-  pull_request:
-    branches:
-      - main
+  workflow_call:
```

Comment:
Don't we want the pipeline to run on PRs?
This change will make the whole pipeline manually driven.

Reply:
Perhaps leave the other events but only publish on `workflow_call`?